### PR TITLE
Multiple network support

### PR DESCRIPTION
**Pull request by AugustoL**
Most important changes are in `src/stores/config` and `src/provider/connectors`

- Adds options to support a set of network, the can be mainnet, ropsten, kovan or develop.
- Uses first network in the supported networks array as default network.
- Config Store now gets the address of the DAT contract based on connected network.
- Replaced fetching of active DAT address form config store in the whole app.
- Small changes in scripts to support multiple network configurations.
- Adds .env.example.

### CODE REVIEW

**Comment by tspoff** [comment]
I'm getting crashes on wrong network, can anyone reproduce?

**Comment by AugustoL** [comment]
@tspoff @pimato Now is showing the connect to main etheruem network message, I fixed conflicts too. Ready for another review.

**Comment by pimato** [comment]
just tested it with "Mainnet, Kovan" and it seems like I am getting this message:
Screenshot 2020-04-28 at 11 33 02 on an **supported network**

Steps to reproduce:

1) yarn start with Mainnet and Kovan
2) connect to Mainnet first
3) switch to Ropsten
4) try to connect and get "please connect to main"
5) switch to Kovan and I can`t connect because I get "please connect to main" message

**Comment by AugustoL** [comment]
@pimato should be fixed now.

**Comment by pimato** [comment]
@AugustoL the button is not switching from `Connect Wallet` to `Wrong Network` after trying to connect via metamask to Ropsten:

Screenshot 2020-04-28 at 16 25 22

**Comment by pimato** [comment]
Just tried to connect to wallet connect on the main network:
Screenshot 2020-04-28 at 21 31 04
seems like wallet connect things it is on **kovan**

**Comment by pimato** [comment]
Also it seems like only the graph has issues when changing from main to kovan:
Maybe we can fix it?
Screenshot 2020-04-28 at 21 47 24

File: src/components/BondingCurveChart.tsx [hunk 1]
```diff
@@ -76,14 +76,16 @@ const BondingCurveChart = observer(({}) => {
     const {
         root: { tradingStore, tokenStore, configStore, datStore },
     } = useStores();
+    
+    const activeDATAddress = configStore.getDXDTokenAddress()
```
Comment (jpkcambridge) [inline src/components/BondingCurveChart.tsx]: I believe the DAT contract is the ERC20 token, but maybe we should make that clear here or somewhere what the connection is between "DAT Address" and "DXD Token Address"

Comment (AugustoL) [reply src/components/BondingCurveChart.tsx]: Yes, we have to create an issue for that, the naming is confusing, this PR is already too big for that I think.

**Comment by AugustoL** [comment]
> Also it seems like only the graph has issues when changing from main to kovan:
> Maybe we can fix it?
> Screenshot 2020-04-28 at 21 47 24
> Screenshot 2020-04-28 at 21 48 53

This is related to how we fetch information form the blokchain, doesnt it shows the right values once the next fecth after the network change is done?

To reproduce this you have to change networks in the right time.

This PR is more dev related than user experience, the user wont be changing network that much.

**Comment by pimato** [comment]
> > Also it seems like only the graph has issues when changing from main to kovan:
> > Maybe we can fix it?
> > Screenshot 2020-04-28 at 21 47 24
> > Screenshot 2020-04-28 at 21 48 53
> 
> This is related to how we fetch information form the blokchain, doesnt it shows the right values once the next fecth after the network change is done?
> 
> To reproduce this you have to change networks in the right time.
> 
> This PR is more dev related than user experience, the user wont be changing network that much.

The rest of the application got fetched without any issues. I am sure we can get that fixed ....

**Pull request merged**